02 - Community Expansions/02.05 All Community Expansions/Themes/ITS Theme.md

```diff
@@ -35,17 +35,6 @@ Modes: [[Dark-mode themes|dark]], [[Light-mode themes|light]]
 
 ## Features
 
-- Features of **ITS Theme** available as snippets:
```

Comment:
This we usually revert since they were manually added (not sure if future commits will fix this)